Doc/library/importlib.rst

```diff
@@ -792,6 +792,96 @@ ABC hierarchy::
         itself does not end in ``__init__``.
 
 
+:mod:`importlib.metadata` -- Metadata
+-------------------------------------
+
+.. module:: importlib.metadata
+    :synopsis: Installed package metadata reading
+
+**Source code:** :source:`Lib/importlib/metadata.py`
+
+--------------
+
+.. versionadded:: 3.8
+
+This module leverages Python's import system to provide access to installed
+package metadata, such as the package's version, entry points, and more.
+
+.. note::
+
+   This module provides functionality similar to `pkg_resources
+   <https://setuptools.readthedocs.io/en/latest/pkg_resources.html>`_ `entry
+   point API
+   <https://setuptools.readthedocs.io/en/latest/pkg_resources.html#entry-points>`_
+   and `metadata API
+   <https://setuptools.readthedocs.io/en/latest/pkg_resources.html#metadata-api>`_.
+
+   The standalone backport of this module provides more information
+   on `using importlib.metadata
+   <https://importlib-metadata.readthedocs.io/en/latest/using.html>`_.
+
+By *installed package* we generally mean a third party package installed into
+Python's ``site-packages`` directory via tools such as ``pip``.  Specifically,
+it means a package with either a discoverable ``dist-info`` or ``egg-info``
+directory, and metadata defined by :PEP:`566` or its older specifications.  By
+default, package metadata can live on the file system or in wheels on
+``sys.path``.  Through an extension mechanism, the metadata can live almost
+anywhere.
+
+Support is provided by special finders on :attr:`sys.meta_path` (see below).
+By default, packages installed on the file system and in :PEP:`491` wheel
+files are supported.
+
+The following exception is defined.
+
+.. exception:: PackageNotFoundError
+
+    Raised when the named package's metadata could not be found.
+
+The following functions are available.
+
+.. function:: entry_points(package):
+
+    Returns the `entry points
+    <https://packaging.python.org/specifications/entry-points/>`_ for the
+    given package, as a :class:`configparser.ConfigParser` instance.  The
+    sections in this ``ConfigParser`` exactly match the sections in the
```

Comment:
:class:`~configparser.ConfigParser`

Reply:
We'll want to revisit this on the next port, as the upstream package doesn't have links to stdlib modules.

Reply:
Turns out this isn't mentioned in the docs anymore either.